src/caja-application.c

```diff
@@ -2143,30 +2216,18 @@ caja_application_local_command_line (GApplication *application,
         files[1] = NULL;
     }
 
-    /*Set up geometry and --browser options  */
+    /*Set up --geometry, --browser and --tabs options  */
     /*Invoke "Open" to create new windows */
-
-    if (browser_window == TRUE && self->priv->geometry == NULL){
-
-        if (len > 0)  {
-        g_application_open (application, files, len, "browser");
+    if (len > 0)  {
+        gchar* concatOptions = g_malloc0(64);
+        if(self->priv->geometry==NULL){
```

Comment:
sometimes you are using a space and sometimes not :)
`if(self->priv->geometry==NULL){` --> `if (self->priv->geometry==NULL) {`

Reply:
and the lines below